alphagov/paas-elasticache-broker
# [#177219242] Apply cost allocation tags

What
---
Applies cost allocation tag `chargeable_entity` to taggable resources created by the broker. This will allow us to work out how much each instance is costing us.

How to review
---
1. Check I got the tag name and value right
1. Make sure there aren't resources I missed
